docs/fairness-indicators.md

```diff
@@ -34,7 +34,7 @@ Run the `demo.py` utility to write the summary results in the log directory:
 
 `python demo.py --logdir=<logdir>/demo --eval_result_output_dir=<eval_result_dir>`
 
-Run TensorBoard:
+Run TensorBoard (Please run the TensorBoard from the same directory in which you have downloaded the evaluation results):
```

Comment:
This is generally not a correct statement. Is this required now? Why?

Reply:
As discussed, this is only for demo purpose.

Reply:
@stephanwlee Ping, let us know if this is OK, or if we should run in a different directory. Thanks!

Reply:
It is obsolete since you have updated the demo section.